pkg/file/tarutil.go

```diff
@@ -40,7 +54,7 @@ func ReaderFromTar(reader io.ReadCloser, tarPath string) (io.ReadCloser, error)
 			}, nil
 		}
 	}
-	return nil, fmt.Errorf("file %s not found in tar", tarPath)
+	return nil, ErrFileNotFound
```

Comment:
Just curious — would it be beneficial here to include the file that wasn't found (e.g. from `tarPath`) in the error? (Which would promote `ErrFileNotFound` to its own error type if we want to avoid abusing error strings)

Reply:
I had been using this error type in testing, then it was removed, I'll revert the change altogether here.

Reply:
but to answer your question, yes, this would lean towards an error type in this case. (Would rather not double click on that in this PR)

Reply:
jk, due to https://github.com/anchore/stereoscope/pull/40#discussion_r493016378 I'll be adding this back. I'll shoot for a local error type.